public/request/user/recalculate-score.php

```diff
@@ -5,13 +5,19 @@
 require_once __DIR__ . '/../../../vendor/autoload.php';
 require_once __DIR__ . '/../../../lib/bootstrap.php';
 
-if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::Registered)) {
+$targetUser = requestInputPost('u');
+
+if (!authenticateFromCookie($actingUser, $permissions, $actingUserDetails, Permissions::Registered)) {
     header("Location: " . getenv('APP_URL') . "?e=badcredentials");
     exit;
 }
 
-if (recalculatePlayerPoints($user)) {
-    header("Location: " . getenv('APP_URL') . "/controlpanel.php?e=recalc_ok");
+if (recalculatePlayerPoints($targetUser)) {
+    if ($targetUser !== $actingUser && $permissions >= Permissions::Admin) {
+        header("Location: " . getenv('APP_URL') . "/user/$targetUser?e=recalc_ok");
```

Comment:
Recommend `e=ok`. `e=recalc_ok` just looks weird when you're redirected to the user's profile:
> Score recalculated! Your new score is shown at the top-right next to your avatar.